tests/core/shh-module/test_shh_filter.py

```diff
@@ -4,24 +4,25 @@
 def test_shh_filter(web3, skip_if_testrpc):
     skip_if_testrpc(web3)
     recieved_messages = []
-    shh_filter = web3.shh.filter({"topics": [web3.fromAscii("test")]})
+    topic = web3.toHex(text="test")
+    shh_filter = web3.shh.filter({"topics": [topic]})
     shh_filter.watch(recieved_messages.append)
 
     payloads = []
     payloads.append(str.encode("payload1"))
     web3.shh.post({
-        "topics": [web3.fromAscii("test")],
-        "payload": web3.fromAscii(payloads[len(payloads) - 1]),
+        "topics": [topic],
+        "payload": web3.toHex(text=payloads[-1]),
```

Comment:
Doesn't need to be cleaned up here, but it might be wise for us to remove the use of `web3.toHex` here in favor of using the `eth_utils.encode_hex` function so as to not make our tests dependent on this API.